### PR TITLE
tests passing

### PR DESCRIPTION
**Pull request by killerwails**
added travis tests

### CODE REVIEW

**Comment by noqcks** [comment]
tests look :+1: @kkwoker can you review the other code?

**Comment by kkwoker** [comment]
Looks pretty good

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Tunables
 --------
-* None
+* influxdb_version: 0.9.6.1
```
Comment (noqcks) [inline README.md]: `influxdb_runtime_root` not documented

Comment (killerwails) [inline README.md]: doesnt have to be. I dont think we want ppl tuning that

File: handlers/main.yml [hunk 2]
```diff
@@ -1,7 +1,14 @@
 ---
-- name: Reload Service | influxdb
+- name: service | influxdb | reloaded
   service:
     state: reloaded
-    name: influxdb
+    name: nosql
```
Comment (noqcks) [inline handlers/main.yml]: these service handlers won't work. Assuming [this line](https://github.com/telusdigital/ansible-influxdb/pull/4/files#diff-468007bd6f59ca63e12c1a3613323a17R22) is correct, then this line shouldn't work.

File: README.md (hunk 1 above)
Comment (noqcks) [inline README.md]: It doesn't _have_ to be, but tunables should be documented whether we toggle them or not imo

**Comment by noqcks** [comment]
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Pull request merged**